rig/place_and_route/place/hilbert.py

```diff
@@ -117,7 +117,8 @@ def place(vertices_resources, nets, machine, constraints):
             # Place the vertex
             unplaced_vertices.remove(constraint.vertex)
             placements[constraint.vertex] = loc
-        elif isinstance(constraint, ReserveResourceConstraint):
+        elif isinstance(constraint,  # pragma: no branch
```

Comment:
That's fine! Its an annoying thing about branch coverage :/ if you have nested branchables which all end at once, there are a huge number of possible branches to cover and it is rare that all of them are meaningfully different.